ups/eupspkg.cfg.sh

```diff
@@ -0,0 +1,5 @@
+build() {
+   python setup.py pytest --addopts "metadetect/tests/ metadetect/lsst/tests/test_import.py"
```

Comment:
Using `nosetests` does not obey the conditional skips, so switching to `pytest`.